hoot-js/src/main/cpp/hoot/js/util/DataConvertJs.h

```diff
@@ -140,7 +140,7 @@ inline void toCpp(v8::Handle<v8::Value> v, QStringList& o)
   }
   v8::Handle<v8::Array> arr = v8::Handle<v8::Array>::Cast(v);
 
-  o.reserve(arr->Length());
+  //o.reserve(arr->Length());
```

Comment:
Why comment this out? Either leave it or remove it. I don't think it does any harm.